scripts/profiler/events.py

```diff
@@ -40,111 +39,48 @@ def deserialize(json):
             json["data_types"],
             json["data_descriptions"])
 
+TRACKED_EVENT_FIELDNAMES = ['type_id', 'timestamp', 'data', 'proc_start_time', 'proc_end_time']
```

Comment:
Make this part of TrackedEvent, please. Use staticmethod or define it as constant for class